sam/models/mlp_model.py

```diff
@@ -333,9 +317,6 @@ def predict(
         X_transformed: pd.DataFrame, optional
             The transformed input data, when return_data is True, otherwise None
         """
-        if max(self.predict_ahead) > 0 and y is None:
```

Comment:
Can we safely remove this check?

Reply:
It should, since y is no longer required at all.

Maybe add a check if `use_diff_of_y and y is None`,

Reply:
Yeah, can't hurt